src/version.txt

```diff
@@ -1 +1 @@
-__version__ = "0.0.10"
+__version__ = "0.1.0-rc.1"
```

Comment:
I think we can go ahead and promote this to `0.1.0`.

Reply:
I built an image in staging yesterday with the `rc.1` version, just to get in practice with doing that.  And for once, I incredibly hadn't forgotten about setting the version for Production.  Version finalized in https://github.com/cisagov/pca-gophish-composition-packer/pull/11/commits/e915cb0b03cfda13d6766afefe210304c92671c9.